### ISSUE TITLE
New key with a null value not applied in update

### ISSUE THREAD

**Issue by mark-rushakoff** [substantial/updeep#8]
This test fails because the result is still `{}`.

```
it('assigns null values', () => {
  expect(u({isNull: null}, {})).to.deep.equal({isNull: null});
});
```

**Comment by aaronjensen** [comment]
Thanks for the repro! Fixed in 0.2.3

**Comment by mark-rushakoff** [comment]
Thanks for the quick fix! I'm digging updeep so far, it plays perfectly with Redux.
